### PR TITLE
Crypto Trend React App

### PR DESCRIPTION
**Pull request by rahimialijan**
# Crypto Trend React App

- Add linter setup and clean linter errors
- Add different components for my project
- Use redux and API for this project
- Add style for whole content

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,33 @@
-import "./App.css";
+import { Routes, Route } from 'react-router';
```
Comment (GzimAsani) [inline src/App.js:1]: - [ ] I couldn't find your video presentation, please be reminded that it is mandatory and must fulfill the following requirements:
image

File: src/components/NotFound/NotFound.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import fourOfour from '../../assets/images/error_404.svg';
+
+const NotFound = () => (
+  <div className="category-secion not-found">
+    <img src={fourOfour} alt="404" />
+    <span className="engineer-emoji"> 🕵️‍♂️ </span>
+  </div>
+);
+
+export default NotFound;
```
Comment (GzimAsani) [inline src/components/NotFound/NotFound.js:11]: - Great job creating a 404 page, it makes your app look more professional :+1:

File: src/components/__test__/Home.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      <Provider store={store}>
+        <Home />
+      </Provider>,
+    );
+  });
+
+  act(() => {});
+
+  expect(screen.getByTestId('home-component')).toBeInTheDocument();
+});
```
Comment (GzimAsani) [inline src/components/__test__/Home.test.js]: - [ ] Great job adding tests, but you need to also test
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @rahimialijan                       :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with creating a 404 page and dividing your components up, the code looks clean +1: 

 ## Required Changes 

- [ ] Check for the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (rahimialijan) [reply src/App.js:1]: done

File: src/components/__test__/Home.test.js (hunk 3 above)
Comment (rahimialijan) [reply src/components/__test__/Home.test.js]: done

**Review by amejid — APPROVED** [review]
Hi @rahimialijan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- App functioning well :heavy_check_mark: 
- Changes implemented :heavy_check_mark: 
- Good presentation :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**